index.html

```diff
@@ -0,0 +1,5154 @@
+<!doctype html><html lang="en">
```

Comment:
Please revert the changes in this file, as it is generated and shouldn't be checked in. I'll be cleaning up our `.gitignore` and worfklows in a separate PR that should prevent this in the future.

Reply:
Apologies - was out sick and missed this ping. Thanks for taking care of it. :)